rules/schema_functions.go

```diff
@@ -387,11 +391,8 @@ func NewPercent(params json.RawMessage) (SchemaFunction[openrtb_ext.RequestWrapp
 }
 
 func (p *percent) Call(wrapper *openrtb_ext.RequestWrapper) (string, error) {
-	percValue := 5 //default value
-	if p.value < 0 {
-		percValue = 0
-	}
-	if p.value > 100 {
+	percValue := 0
+	if p.value >= 100 {
 		percValue = 100
 	}
```

Comment:
Oh I think the logic here should be:
```
percValue := p.value
if percValue < 0 {
    percValue = 0
}
if percValue > 100 {
    percValue = 100
}
```

We use the percent value given as an arg but ensure that it is within the range of 0 and 100.